docs/pages/operation/monitoring.mdx

```diff
@@ -167,4 +168,7 @@ avg(erpc_upstream_score_overall) by (network, upstream)
 
 # CORS issues by origin
 sum(rate(erpc_cors_disallowed_origin_total{}[5m])) by (project, origin)
-```
+
+# Multiplexed poll operations by finalized poll operation
+sum(rate(erpc_upstream_multiplexed_polls_total{operation="finalized"}[5m])) by (network, upstream)
```

Comment:
Not sure if this metric is valuable from user's perspective? It creates bunch of cardinality but probably wouldn't help investigating issues? As a erpc core dev though it'll be useful during development.

Reply:
hmm interesting. would the same apply to `erpc_network_multiplexed_request_total ` as well?

Reply:
That one is actually useful as the requests are done by the users and they would get benefit from knowing how many were multiplexed. This internal polling call should be as few as possible from user's PoV anyway, anything more than necessary would be our code logic flaw. In perfect world we don't even need this multiplexer because our locking mechanisms wouldn't even cause too many redundant polls.